src/OpenTelemetry.Contrib.Instrumentation.MassTransit/MassTransitEnrichEventType.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="MassTransitEnrichEventType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+
+namespace OpenTelemetry.Contrib.Instrumentation.MassTransit
+{
+    /// <summary>
+    /// Describes the possible events fired when enriching an <see cref="Activity"/>.
+    /// </summary>
+    public enum MassTransitEnrichEventType
```

Comment:
this is a new public API, but it doesn't seem to be used anywhere?

Reply:
I relied on the implementation of AddWcfInstrumentation.
This can be used via options overload:

```
.AddMassTransitInstrumentation(o =>
{
    o.Enrich = (activity, eventName, rawObject) =>
    {
        if (eventName.Equals("OnStartActivity"))
        {
            var messagingOperation = activity.OperationName.Split('.').Last().ToLower();
            if (messagingOperation != "send")
                activity.SetTag(SemanticConventions.AttributeMessagingOperation, messagingOperation);

            if (rawObject is ConsumeContext consumeContext)
            {
                activity.SetTag("messaging.masstransit.retry_count", consumeContext.GetRetryCount());
            }
        }
    };
})
```